src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -271,28 +271,35 @@ protected function getDefaultRouteName(\ReflectionClass $class, \ReflectionMetho
         return $name;
     }
 
-    protected function getGlobals(\ReflectionClass $class)
+    protected function getGlobals(\ReflectionClass $class): array
```

Comment:
Adding a return type declaration to a protected non-final method is a breaking change. This change has to be reverted.

Feel free to add a doc block though.

Reply:
Indeed
```suggestion
    /**
     * @return array
     */
    protected function getGlobals(\ReflectionClass $class)
```